### PR TITLE
docs: add documentation about product opener infrastructure

### PR DESCRIPTION
**Pull request by raphael0202**

### CODE REVIEW

**Review by alexgarel — COMMENTED** [review]
@raphael0202 I will complete a bit what you have done.

File: docs/product-opener.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+In production, it's deployed on the off2 server (off2.openfoodfacts.org).
+Each flavor of Open Food Facts (OFF, OBF, OPF...) is deployed in its own proxmox VM:
+
+| Flavor | VM |
+| ---- | --- |
+| OPFF | 110 |
+| OBF  | 111 |
+| OPF  | 112 |
+| OFF  | 113 |
```
Comment (stephanegigandet) [inline docs/product-opener.md:16]:
```suggestion
| OFF  | 113 |
| OFF-PRO | 114 |
```

**Comment by raphael0202** [comment]
@alexgarel What about merging this now, if there is nothing factually wrong about it?
IMO it's better to merge it and improve it later than keeping it as an  open PR

**Review by alexgarel — APPROVED** [review]
Thanks

**Pull request merged**